src/Xamarin.Android.Build.Tasks/Xamarin.Android.Build.Tasks.csproj

```diff
@@ -9,7 +9,7 @@
     <RootNamespace>Xamarin.Android.Tasks</RootNamespace>
     <AssemblyName>Xamarin.Android.Build.Tasks</AssemblyName>
     <FileAlignment>512</FileAlignment>
-    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.5.1</TargetFrameworkVersion>
```

Comment:
The whole reason for this change is because Xamarin Studio/Visual Studio for Mac refused to properly build this project, because one of the projects it referenced (libZipSharp?) had a `$(TargetFrameworkVersion)` of v4.5.1.

Which is funny, because it built Just Fine™ from the command-line... :-/

We could remove this line, if necessary, but it would be nice to be able to load/build from XS...